src/nvim/api/buffer.c

```diff
@@ -1081,7 +1081,7 @@ Boolean nvim_buf_is_loaded(Buffer buffer)
 /// @param opts  Optional parameters. Keys:
 ///          - force:  Force deletion and ignore unsaved changes.
 ///          - unload: Unloaded only, do not delete. See |:bunload|
-void nvim_buf_delete(Buffer buffer, Dictionary opts, Error *err)
+void nvim_buf_delete(Buffer buffer, Dict(delete_opts) *opts, Error *err)
```

Comment:
Minor nitpick (and some guideline), `buf_delete_opts` is a better name for the keyset imo. Generally, I think you can just name the keyset by removing the `nvim_` part from the API function name, that works in most cases.

Reply:
Renamed to `Dict(buf_delete)`.